src/Networking/NexusMods.Networking.Downloaders/DownloadService.cs

```diff
@@ -143,6 +143,14 @@ public Optional<Percent> GetTotalProgress()
         return Optional<Percent>.Create(Percent.CreateClamped(total / tasks.Length));
     }
 
+    /// <inheritdoc />
+    public async Task SetIsHidden(bool isHidden, IDownloadTask[] targets)
+    {
+        var tx = _conn.BeginTransaction();
+        await Task.WhenAll(targets.Select(x => x.SetIsHidden(isHidden, tx)));
```

Comment:
See suggestion in `ADownloadTask`.

```suggestion
        foreach (var downloadTask in targets)
        {
            downloadTask.SetIsHidden(isHidden, tx);
        }
```